### PR TITLE
Editorial, and QA on charter.html

### PR DESCRIPTION
**Pull request by koalie**

### CODE REVIEW

File: process/charter.html [hunk 1]
```diff
@@ -470,20 +468,20 @@ <h4><a name="join-form">After sending the W3C Decision, Call for Participation</
 <p>After sending the W3C Decision and Call for Participation:</p>
 
 <ul>
-<li>Make sure that IPP has the latest charter link (the W3C Communications Team <a href="https://www.w3.org/admin/dashboard">searches</a> the group and edits information to add a new charter)
+<li>Make sure that Groups DB has the latest charter link (the W3C Communications Team <a href="/admin/dashboard">searches</a> the group and edits information to add a new charter).
```
Comment (csarven) [inline process/charter.html:471]: Can it be expanded (to dashboard? database? something else?), or use `abbr`, or link to DB?

File: process/charter.html [hunk 2]
```diff
@@ -214,52 +214,52 @@ <h3 id="recharter-request">Request for rechartering</h3>
...
+      <li>Work with the W3C Communications Team to send <a href="adv-notice.html">advance notice to the AC</a> as soon as you have something to point to, and update the <a href="#pipeline">pipeline</a>.</li>
       <li>Start the horizontal reviews of the charter as soon as the scope, deliverables, and dependencies
          parts of the charter are stable enough. Ping the horizontal reviewers on a weekly basis.</li>
-      <li>Start working on resolving any formal objection on a charter as soon as those received. Don't wait
-        for the end of the AC review period.
+      <li>Start working on resolving Formal Objections to a charter as soon as those are received. Don't wait
+        till the end of the AC review period.
       </li>
-      <li>Prepare a draft of your proposed W3C approval announcement before getting the approval from TiLT</li>
+      <li>Prepare a draft of the W3C Desicion announcement before getting the approval from TiLT</li>
```
Comment (csarven) [inline process/charter.html:254]:
```suggestion
      <li>Prepare a draft of the W3C Decision announcement before getting the approval from TiLT</li>
```

Comment (csarven) [reply process/charter.html:254]: Also, can "W3C Decision" be more precise e.g., whether that's W3C *Team Decision* (as per Process) or W3C-wide (including Team, Members, AB, ..)? The context is not entirely clear to me, i.e., is it related to the responses to the FO or other change requests or a final decision.

**Review by csarven — CHANGES_REQUESTED** [review]

File: process/charter.html (hunk 1 above)
Comment (koalie) [reply process/charter.html:471]: “Groups DB” is our group’s database, not “dashboard”.

File: process/charter.html (hunk 2 above)
Comment (koalie) [reply process/charter.html]: It’s the “W3C Decision” as defined in the Process (so one that results from Members review through a questionnaire).

Comment (csarven) [reply process/charter.html]: I completely overlooked that, thanks!

File: process/charter.html (hunk 1 above)
Comment (csarven) [reply process/charter.html:471]: Thanks. I updated the suggestion but I'll yield. I take it that those that know / have access to "Groups DB" know what it is. I was just reading it as an outsider and it wasn't clear to me.

Comment (koalie) [reply process/charter.html:471]: It’s in the section three that documents for transparency what the team does in the sausage factory.

**Pull request merged**